doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -871,6 +871,18 @@ Quitting and debugging
    message in the place of the instruction count.
 
 
+.. cmd:: Profile {? @string } @sentence
+
+   Executes :n:`@sentence` and displays profiling information. If
+   :n:`@string` is given, a full trace is written to
+   ":n:`@string`.json".
+
+   If :n:`@string` is a relative filename, it refers to the directory
+   specified by the command line option `-output-directory`, if set
+   (see :ref:`command-line-options`) and otherwise, the current
```

Comment:
```suggestion
   specified by the :ref:`command line option <command-line-options`> `-output-directory`, if set
   and otherwise, the current
```

A little easier to read without the parenthetical comment.  You could also define a reference point for `-output-directory` and put the link on `-output-directory`, though probably without the font change from the ``` ` ... ` ```.  Alternative: move the "(see ref ...)" after the end of the sentence.